pubtools/_quay/clear_repo.py

```diff
@@ -121,6 +122,12 @@ def clear_repositories(repositories: str, settings: dict[str, Any]) -> None:
     item_processor = item_processor_for_internal_data(
         quay_client, "quay.io", 5, settings["quay_org"]
     )
+
+    signer_settings = {k: v for k, v in settings.items() if k not in ["quay_org"]}
+    signer_settings["quay_namespace"] = settings["quay_org"]
+    signer_settings["dest_quay_api_token"] = os.environ.get("QUAY_API_TOKEN")
```

Comment:
Why is the transformation necessary? Is there danger that the token may be displayed in pub logs?